num_to_words/utils/constants.py

```diff
@@ -1,1106 +1,1219 @@
-    "77": "ಎಪ್ಪತ್ತೇಳು",
-    "78": "ಎಪ್ಪತ್ತೆಂಟು",
-    "79": "ಎಪ್ಪತ್ತೊಂಬತ್ತು",
-    "80": "ಎಂಬತ್ತು",
-    "81": "ಎಂಬತ್ತೊಂದು",
-    "82": "ಎಂಬತ್ತೆರಡು",
-    "83": "ಎಂಬತ್ತ್ ಮೂರು",
-    "84": "ಎಂಬತ್ತ್ ನಾಲ್ಕು",
-    "85": "ಎಂಬತ್ತೈದು",
-    "86": "ಎಂಬತ್ತಾರು",
-    "87": "ಎಂಬತ್ತೇಳು",
-    "88": "ಎಂಬತ್ತೆಂಟು",
-    "89": "ಎಂಬತ್ತೊಂಬತ್ತು",
-    "90": "ತೊಂಬತ್ತು",
-    "91": "ತೊಂಬತ್ತೊಂದು",
-    "92": "ತೊಂಬತ್ತೆರಡು",
-    "93": "ತೊಂಬತ್ತ ಮೂರು",
-    "94": "ತೊಂಬತ್ತ ನಾಲ್ಕು",
-    "95": "ತೊಂಬತ್ತೈದು",
-    "96": "ತೊಂಬತ್ತಾರು",
-    "97": "ತೊಂಬತ್ತೇಳು",
-    "98": "ತೊಂಬತ್ತೆಂಟು",
-    "99": "ತೊಂಬತ್ತೊಂಬತ್ತು",
-    "100": "ನೂರ",
-    "1000": "ಸಾವಿರದ",
-    "100000": "ಲಕ್ಷದ",
-    "10000000": "ಕೋಟಿ",
-    "1000000000": "ಶತಕೋಟಿ",
+	"0": "ಸೊನ್ನೆ",
+	"1": "ಒಂದು",
+	"2": "ಎರಡು",
+	"3": "ಮೂರು",
+	"4": "ನಾಲ್ಕು",
+	"5": "ಐದು",
+	"6": "ಆರು",
+	"7": "ಏಳು",
+	"8": "ಎಂಟು",
+	"9": "ಒಂಬತ್ತು",
+	"10": "ಹತ್ತು",
+	"11": "ಹನ್ನೊಂದು",
+	"12": "ಹನ್ನೆರಡು",
+	"13": "ಹದಿಮೂರು",
+	"14": "ಹದಿನಾಲ್ಕು",
+	"15": "ಹದಿನೈದು",
+	"16": "ಹದಿನಾರು",
+	"17": "ಹದಿನೇಳು",
+	"18": "ಹದಿನೆಂಟು",
+	"19": "ಹತ್ತೊಂಬತ್ತು",
+	"20": "ಇಪ್ಪತ್ತು",
+	"21": "ಇಪ್ಪತ್ತೊಂದು",
+	"22": "ಇಪ್ಪತ್ತೆರಡು",
+	"23": "ಇಪ್ಪತ್ತಮೂರು",
+	"24": "ಇಪ್ಪತ್ತನಾಲ್ಕು",
+	"25": "ಇಪ್ಪತ್ತೈದು",
+	"26": "ಇಪ್ಪತ್ತಾರು",
+	"27": "ಇಪ್ಪತ್ತೇಳು",
+	"28": "ಇಪ್ಪತ್ತೆಂಟು",
+	"29": "ಇಪ್ಪತ್ತೊಂಬತ್ತು",
+	"30": "ಮೂವತ್ತು",
+	"31": "ಮೂವತ್ತೊಂದು",
+	"32": "ಮೂವತ್ತೆರಡು",
+	"33": "ಮುವತ್ತ ಮೂರು",
+	"34": "ಮೂವತ್ತು ನಾಲ್ಕು",
+	"35": "ಮೂವತ್ತೈದು",
+	"36": "ಮೂವತ್ತಾರು",
+	"37": "ಮೂವತ್ತೇಳು",
+	"38": "ಮೂವತ್ತು ಎಂಟು",
+	"39": "ಮೂವತ್ತು ಒಂಬತ್ತು",
+	"40": "ನಲವತ್ತು",
+	"41": "ನಲವತ್ತೊಂದು",
+	"42": "ನಲವತ್ತು ಎರಡು",
+	"43": "ನಲವತ್ಮೂರು",
+	"44": "ನಲವತ್ತು ನಾಲ್ಕು",
+	"45": "ನಲವತ್ತೈದು",
+	"46": "ನಲವತ್ತಾರು",
+	"47": "ನಲವತ್ತೇಳು",
+	"48": "ನಲವತ್ತೆಂಟು",
+	"49": "ನಲವತ್ತೊಂಬತ್ತು",
+	"50": "ಐವತ್ತು",
+	"51": "ಐವತ್ತೊಂದು",
+	"52": "ಐವತ್ತೆರಡು",
+	"53": "ಐವತ್ತು ಮೂರು",
+	"54": "ಐವತ್ನಾಲ್ಕು",
+	"55": "ಐವತ್ತೈದು",
+	"56": "ಐವತ್ತಾರು",
+	"57": "ಐವತ್ತೇಳು",
+	"58": "ಐವತ್ತೆಂಟು",
+	"59": "ಐವತ್ತೊಂಬತ್ತು",
+	"60": "ಅರವತ್ತು",
+	"61": "ಅರವತ್ತೊಂದು",
+	"62": "ಅರವತ್ತೆರಡು",
+	"63": "ಅರವತ್ತಮೂರು",
+	"64": "ಅರವತ್ನಾಲ್ಕು",
+	"65": "ಅರವತ್ತೈದು",
+	"66": "ಅರವತ್ತಾರು",
+	"67": "ಅರವತ್ತೇಳು",
+	"68": "ಅರವತ್ತೆಂಟು",
+	"69": "ಅರವತ್ತೊಂಬತ್ತು",
+	"70": "ಎಪ್ಪತ್ತು",
+	"71": "ಎಪ್ಪತ್ತೊಂದು",
+	"72": "ಎಪ್ಪತ್ತೆರಡು",
+	"73": "ಎಪ್ಪತ್ತಮೂರು",
+	"74": "ಎಪ್ಪತ್ನಾಲ್ಕು",
+	"75": "ಎಪ್ಪತ್ತೈದು",
+	"76": "ಎಪ್ಪತ್ತಾರು",
+	"77": "ಎಪ್ಪತ್ತು ಏಳು",
+	"78": "ಎಪ್ಪತ್ತು ಎಂಟು",
+	"79": "ಎಪ್ಪತ್ತೊಂಭತ್ತು",
+	"80": "ಎಂಬತ್ತು",
+	"81": "ಎಂಬತ್ತೊಂದು",
+	"82": "ಎಂಬತ್ತೆರಡು",
+	"83": "ಎಂಬತ್ಮೂರು",
+	"84": "ಎಂಭತ್ತ ನಾಲ್ಕು",
+	"85": "ಎಂಬತ್ತೈದು",
+	"86": "ಎಂಭತ್ತಾರು",
+	"87": "ಎಂಬತ್ತೇಳು",
+	"88": "ಎಂಬತ್ತೆಂಟು",
+	"89": "ಎಂಬತ್ತೊಂಬತ್ತು",
+	"90": "ತೊಂಬತ್ತು",
+	"91": "ತೊಂಬತ್ತೊಂದು",
+	"92": "ತೊಂಬತ್ತೆರಡು",
+	"93": "ತೊಂಬತ್ಮೂರು",
+	"94": "ತೊಂಭತ್ನಾಲ್ಕು",
+	"95": "ತೊಂಬತ್ತೈದು",
+	"96": "ತೊಂಬತ್ತಾರು",
+	"97": "ತೊಂಬತ್ತೇಳು",
+	"98": "ತೊಂಬತ್ತೆಂಟು",
+	"99": "ತೊಂಬತ್ತೊಂಬತ್ತು",
+	"100": "ನೂರು",
+	"1000": "ಸಾವಿರ",
+	"100000": "ಲಕ್ಷದ",
+	"10000000": "ಕೋಟಿ",
+	"10,00000000": "ಶತಕೋಟಿ
```

Comment:
no comma here please
```suggestion
	"1000000000": "ಶತಕೋಟಿ
```